### ISSUE TITLE
[SDR.handleFiles] Found class name dirs: []

### ISSUE THREAD

**Issue by a6001663** [square/spoon#413]
I am using spoon 1.7.0. During the execution from jenkins, i see this message [SDR.handleFiles] Found class name dirs: []. I have added the debug key as suggested by someone on one of the forums. I cant seem to get away from this to be able to use spoon.

**Comment by lifuzu** [comment]
+1 I have the same issue, any update?

**Comment by kwiky** [comment]
+1

**Comment by Imran-ss** [comment]
+1 same issue I am facing `[SDR.handleFiles] Found class name dirs: []`.
Any update ...

**Comment by a6001663** [comment]
No, not sure what I did that I don’t see this error anymore. Ill go back and talk to the team if they remember anything about it.


Ankit Agarwal
Release Engineer
Best Buy Co., Inc. | Technology Development Center
Mobile: (317)654-6900
Email: Ankit.Agarwal@BestBuy.com<mailto:Ankit.Agarwal@BestBuy.com>

Quality is never an accident, it is always the result of intelligent effort- John Ruskin

**Comment by quiro91** [comment]
I'm experiencing this too in Spoon 2.0.0-SNAPSHOT

**Comment by jrbacon** [comment]
I've found that this issue arises when a non-debug apk is used in testing.  Switching to a debug version solved the problem for me.
